src/docs/product/performance/filters-display.mdx

```diff
@@ -67,10 +56,36 @@ Widgets offer visualizations that you can change to best match your workflow.
 
 In the first row, there are time series and distribution graphs of common metrics for the selected project type (frontend or backend), such as User Misery or Failure Rate. You can compare these side-by-side to find any correlations.
 
-In the second row, there are actionable widgets that can show you areas of concern in your application, such as Most Related Errors or Most Recurring Issues. Below these widgets, there are links to other parts of the application, which you can follow to see additional context.
+In the second row, there are actionable widgets that can show you areas of concern in your application, such as Most Regressed Transactions or Most Recurring Issues. Below these widgets, there are links to other parts of the application, which you can follow to see additional context. Transactions shown in these lists can be temporarily removed from the view by clicking the 'X' beside the transaction name.
```

Comment:
Not sure if we need to mention the ability to remove the transaction from the list, there is already a tooltip so maybe it's fine to elide this.

Reply:
I think the fact that the removal is temporary is helpful to point out, so I'd keep it in.